lib/player.js

```diff
@@ -5790,9 +5790,8 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
         hdrLevel: '',
         spatialAudio: false,
         videoLayout: '',
-        label,
         videoLabel: '',
-        audioLabel: '',
+        audioLabel: label || '',
```

Comment:
According to types, label should be always defined string, so we can simplify here
```suggestion
        audioLabel: label,
```